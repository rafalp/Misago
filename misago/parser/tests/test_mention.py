def test_mention(parse_markup):
    result = parse_markup("Hello @Bob_Boberston!")
    assert result == [
        {
            "type": "paragraph",
            "children": [
                {"type": "text", "text": "Hello "},
                {
                    "type": "mention",
                    "username": "Bob_Boberston",
                },
                {"type": "text", "text": "!"},
            ],
        }
    ]
