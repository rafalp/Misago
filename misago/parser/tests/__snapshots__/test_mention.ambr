# serializer version: 1
# name: test_mention_for_inactive_user
  '<p>this <span misago-mention="inactive-user" class="rich-text-mention-not-found">@Inactive_User</span> is inactive</p>'
# ---
# name: test_mention_for_not_found_user
  '<p>this <span misago-mention="user" class="rich-text-mention-not-found">@User</span> doesnâ€™t exist</p>'
# ---
